src/environment.cpp

```diff
@@ -1322,9 +1324,8 @@ void ServerEnvironment::step(float dtime)
 	/*
 		Manage active block list
 	*/
-	if(m_active_blocks_management_interval.step(dtime, 2.0))
-	{
-		ScopeProfiler sp(g_profiler, "SEnv: manage act. block list avg /2s", SPT_AVG);
+	if (m_active_blocks_management_interval.step(dtime, m_cache_abm_interval)) {
+		ScopeProfiler sp(g_profiler, "SEnv: manage act. block list avg /interval", SPT_AVG);
```

Comment:
names are just a suggestion, of course. could be active_block_mgmt_interval as well, or similar